gcloud/pubsub/client.py

```diff
@@ -50,6 +50,12 @@ def list_topics(self, page_size=None, page_token=None):
         See:
         https://cloud.google.com/pubsub/reference/rest/v1/projects.topics/list
 
+        Example:
+
+        .. literalinclude:: pubsub_snippets.py
+           :start-after: [START client_list_topics]
+           :end-before: [END client_list_topics]
```

Comment:
> Embedding snippets in the method docstrings makes testing them infeasible

You're saying that we couldn't do the code shuffling the other way around (parse the Docstring and shove the raw text into a Python file which is then tested) ? That doesn't seem right to me.

I can see if the argument is it's inconvenient, or ugly, or maybe impractical, but impossible?